build-ios.sh

```diff
@@ -23,14 +23,14 @@ IOS_ARCH="x86_64"
 fi
 
 # Set Platform: device or simulator
-if [ "$2" == "device" ] || [ "$3" == "device" ]; then
+if [ "$2" == "device" ] || [ "$3" == "device" ] || [ "$4" == "device" ]; then
 IOS_PLAT="iphoneos"
 else
 IOS_PLAT="iphonesimulator"
 fi
 
 # Set target iOS minver
-IOS_DEPLOYMENT_TARGET=10.0
+IOS_DEPLOYMENT_TARGET=14.0
```

Comment:
Why was this changed?

Reply:
Will add another PR to pass ios deployment target as argument.